### PR TITLE
Feature tabular invalid data

### PR DESCRIPTION
**Pull request by bewakes**

### CODE REVIEW

File: src/components/TabularSheet/renderers.js [hunk 1]
```diff
@@ -15,9 +18,8 @@ const cellDefaultProps = {
     options: {},
 };
 
-// eslint-disable-next-line no-unused-vars
-export const StringCell = ({ value, className, options }) => (
-    <div className={className}>
+export const StringCell = ({ value, className, options: { invalid = false } }) => (
+    <div className={_cs(className, invalid ? styles.invalid : '')}>
```
Comment (tnagorra) [inline src/components/TabularSheet/renderers.js]: Just use ```_cs(className, invalid && styles.invalid)```

File: src/components/TabularSheet/renderers.js [hunk 2]
```diff
@@ -55,3 +57,10 @@ export const DateCell = ({ value, className, options }) => (
 
 DateCell.propTypes = cellPropTypes;
 DateCell.defaultProps = cellDefaultProps;
+
+export const InvalidHandledCell = Cell => ({ value, className, options }) => {
```
Comment (tnagorra) [inline src/components/TabularSheet/renderers.js]: Decorators should start with lowercase verb

File: src/views/LeadAdd/LeadFormItem/index.js [hunk 3]
```diff
@@ -363,6 +352,36 @@ export default class LeadFormItem extends React.PureComponent {
         const type = leadAccessor.getType(lead);
         const values = leadAccessor.getFaramValues(lead);
 
+        if (values.tabularBook) {
```
Comment (tnagorra) [inline src/views/LeadAdd/LeadFormItem/index.js]: This block should have been removed.

File: src/components/TabularSheet/renderers.js (hunk 1 above)
Comment (bewakes) [reply src/components/TabularSheet/renderers.js]: fixed

File: src/views/LeadAdd/LeadFormItem/index.js (hunk 3 above)
Comment (bewakes) [reply src/views/LeadAdd/LeadFormItem/index.js]: fixed

**Pull request merged**